### PR TITLE
RabbitMQ monitoring (#1487)

### PR DESCRIPTION
**Pull request by sbbroot**
* Add Grafana dashboards for RabbitMQ monitoring

### CODE REVIEW

**Review by atsikham — COMMENTED** [review]
Looks good, but conflicts need to be resolved.

File: core/src/epicli/data/common/defaults/configuration/grafana.yml [hunk 1]
```diff
@@ -91,6 +94,9 @@ specification:
     # - dashboard_id: '358'
     #   revision_id: '1'
     #   datasource: 'Prometheus'
+    # - dashboard_id: '10991'
+    #   revision_id: '11'
+    #   datasource: 'Prometheus'
```
Comment (rafzei) [inline core/src/epicli/data/common/defaults/configuration/grafana.yml]: It's actually useless here (it's just an example)

**Review by rafzei — CHANGES_REQUESTED** [review]
Please update table with license for this dashboard, [here](https://github.com/epiphany-platform/epiphany/blob/develop/docs/home/COMPONENTS.md#predefined-grafana-dashboards)

**Review by seriva — APPROVED** [review]

**Review by erzetpe — APPROVED** [review]

**Review by rafzei — APPROVED** [review]

**Comment by seriva** [comment]
Merged as per @przemyslavic  permission as we want todo #2643 ASAP. He will test it from develop.

**Pull request merged**